### ISSUE TITLE
[QA] generate a11y accessibility report for cucumber tests

### ISSUE THREAD

**Issue by nirajacharya2** [owncloud/web#13366]
When working on a feature the dev needs to take in mind about the a11y test coverage. Also anybody working on the tests should keep in mind for the a11 coverage. We already have report. Report is working only for Playwright, not working for Cucumber yet.

```
📊 Accessibility Report Generated:
   Violations: 0
   File: /var/www/owncloud/web/reports/e2e/a11y-report.json
   Tests: 1
✅ No accessibility violations found
```
